packages/roosterjs-editor-dom/lib/selection/selectTable.ts

```diff
@@ -0,0 +1,158 @@
+import { SelectionRangeTypes, TableSelection, TableSelectionRange } from 'roosterjs-editor-types';
+
+const TABLE_ID = 'roosterjs_tableSelected_';
+const STYLE_TAG_ID = 'style_';
+
+/**
+ * Select a table
+ * @param table table to select
+ * @param coordinates first and last cell of the selection, if this parameter is null, instead of
+ * selecting, will unselect the table.
+ * @param document owner document of the table, if added will append a style element with styles to add to the table
+ * @returns TableSelectionRange if selected a new table, if unselecting the table, null
+ */
+export default function selectTable(
```

Comment:
Suggest to put this into core package, and make it a core api, can either integrate together with select, or a separate one